### DISCUSSION TITLE
Nextcloud non-responsive after approx. 13 days uptime

### DISCUSSION THREAD

**Discussion by szaimen** [nextcloud/all-in-one#4198] [Questions] [answered]
I have been having the same issue for the last few months (Nextcloud non-responsive after approx. 13 days uptime).

I have documented my _torturous_ journey [here](https://help.nextcloud.com/t/nextcloud-aio-v7-11-2-non-responsive-high-cpu-every-13-days-network-locationon-host-for-ncdata/178768) at Nextcloud help forum. @

I have had no support from Nextcloud team or recognition of this issue affecting others apart from you, but I was sure I wasn't the only one experiencing this out there. So thanks for sharing @MariusQuabeck!

Issue seemed to start around AIO v7.9.1 and continues to present day on AIO v7.11.2, with a totally different setup to you.
I'm running on:
**Windows 11, with Hyper VM running Ubuntu 22.04.3 LTS, with **Datadir** pointing to the **network location on windows  host**.

**My investigation has shown everything is pointing to root/ncadmin processes stuck on loop with network operations trying to communicate over SMB and using 2-4 cores intensely stopping Nextcloud master container and Ubuntu from communicating internally in private network or externally exposed hostname. There workaround for me was to to keep a ssh connected at the start of the VM start, and keep it open with htop showing.** 
@MariusQuabeck Are you using SMB share as datadir like me?

To note, I have set up as per [AIO documentation](https://github.com/nextcloud/all-in-one?tab=readme-ov-file#how-to-change-the-default-location-of-nextclouds-datadir).

@szaimen I have tried sudo apt update; sudo apt upgrade; sudo reboot the last time this occurred on 24th January, experienced this with Ubuntu 22.04.1 LTS and Ubuntu 22.04.3 LTS. No improvements or change, AIO still hanging after approx. 13 days. I suspect this may not be a docker issue per se, but an issue with networking comms using SMB as Datadir. Please have a look at my post [here](https://help.nextcloud.com/t/nextcloud-aio-v7-11-2-non-responsive-high-cpu-every-13-days-network-locationon-host-for-ncdata/178768) for further detail.

@szaimen Here's my `sudo docker info`:
```
Client: Docker Engine - Community
 Version:    25.0.0
 Context:    default
 Debug Mode: false
 Plugins:
  buildx: Docker Buildx (Docker Inc.)
    Version:  v0.12.1
    Path:     /usr/libexec/docker/cli-plugins/docker-buildx
  compose: Docker Compose (Docker Inc.)
    Version:  v2.24.1
    Path:     /usr/libexec/docker/cli-plugins/docker-compose
  scan: Docker Scan (Docker Inc.)
    Version:  v0.23.0
    Path:     /usr/libexec/docker/cli-plugins/docker-scan

Server:
 Containers: 17
  Running: 14
  Paused: 0
  Stopped: 3
 Images: 19
 Server Version: 25.0.0
 Storage Driver: overlay2
  Backing Filesystem: extfs
  Supports d_type: true
  Using metacopy: false
  Native Overlay Diff: true
  userxattr: false
 Logging Driver: json-file
 Cgroup Driver: systemd
 Cgroup Version: 2
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local splunk syslog
 Swarm: inactive
 Runtimes: io.containerd.runc.v2 runc
 Default Runtime: runc
 Init Binary: docker-init
 containerd version: a1496014c916f9e62104b33d1bb5bd03b0858e59
 runc version: v1.1.11-0-g4bccb38
 init version: de40ad0
 Security Options:
  apparmor
  seccomp
   Profile: builtin
  cgroupns
 Kernel Version: 5.15.0-92-generic
 Operating System: Ubuntu 22.04.3 LTS
 OSType: linux
 Architecture: x86_64
 CPUs: 4
 Total Memory: 10.42GiB
 Name: nextcloud
 ID: 934c1cb8-3402-4ccf-ae66-59d51c9cde6c
 Docker Root Dir: /var/lib/docker
 Debug Mode: false
 Experimental: false
 Insecure Registries:
  127.0.0.0/8
 Live Restore Enabled: false
```

_Originally posted by @sunnyd24 in https://github.com/nextcloud/all-in-one/issues/4183#issuecomment-1932134670_

**Comment by MariusQuabeck** [comment]
I'm answering here to keep the issue on track, this part has been moved to discussion for a reason I assume :)

@sunnyd24 thank you for confirming the issue I was experiencing. it looks like you have provided much more logs and details than I managed to think of, so thank you, I hope this helps solving the issue. I can generally confirm all your findings so this should be the same issue.

I know from my personal and professional experience that @szaimen is incredibly dedicated to AIO and that he will make sure that issues are being addressed correctly. 

> @MariusQuabeck Are you using SMB share as datadir like me?

I'm not sure how the client for macOS works these days and my AIO instance has ~5 users so I can't speak for all of them but I'm not using an SMB share afaik

**Comment by szaimen** [comment, marked as answer]
Hi @sunnyd24, I'll answer your post at https://help.nextcloud.com/t/nextcloud-aio-v7-11-2-non-responsive-high-cpu-every-13-days-network-locationon-host-for-ncdata/178768.

**Reply by szaimen** [reply]
Also see https://github.com/nextcloud/all-in-one/discussions/4327#discussioncomment-8665498
